### DISCUSSION TITLE
Rotating text

### DISCUSSION THREAD

**Discussion by jonmmease** [grovesNL/glyphon#72] [General]
Hi! Thanks for your work on this project.

I was wondering if it's possible today to rotate text. I didn't find any reference to text rotation in the glyphon or cosmic-text docs, but was hoping it's possible somehow.

Thanks!

**Comment by grovesNL** [comment]
Hi! 👋 There's some discussion about it and trying to figure out the right API in #64 and #4 - it would be great to hear how you'd like it to work.

**Reply by jonmmease** [reply]
Thanks for the response! I commented on my use case over in #64, and would be happy to help out with the implementation once we settle on the initial API.
